src/main/java/com/internship/frejaeidjmeterplugin/jmeter/settings/EmailSettings.java

```diff
@@ -0,0 +1,51 @@
+package com.internship.frejaeidjmeterplugin.jmeter.settings;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class EmailSettings {
+
+    private static HashMap<Integer, String> emails;
+    private static List<String> passwords;
+
+    static {
+        emails = new HashMap<>();
```

Comment:
Why do we need emails map also here? I think we have it in MobileClientService class.